cmd/gateway/main.go

```diff
@@ -94,9 +95,15 @@ func main() {
 
 func setupGlobalLogger(env string) (logger *zap.Logger, err error) {
 	if env == "production" {
-		logger, err = zap.NewProduction()
+		zapConfig := zap.NewProductionConfig()
+		zapConfig.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+		logger, err = zapConfig.Build()
 	} else {
-		logger, err = zap.NewDevelopment()
+		zapConfig := zap.NewDevelopmentConfig()
+		zapConfig.Encoding = "json"
```

Comment:
I find JSON in development really hard to read, which makes developing slower.

Taking a step back, what's the use case for sending dev logs to datadog?

Reply:
Logging is only controlled by the development/production flags atm. So to get debug logs we have to turn on development. We might want to do this for debugging issues.

Zap's baked in development config has more than just debug logs. For example it includes stack traces for anything above warn. 

The better solution perhaps is to expose a parameter called log level, and pipe that into the zap logger's config.

Reply:
Gotcha.

Yeah, exposing log level as an env variable makes a lot of sense. No need to always couple the log level to the logging environment.